pkg/grpc/vehicle_signal_decoding.proto

```diff
@@ -7,6 +7,22 @@ option go_package = "github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc";
 
 package grpc;
 
+message PIDConfig {
```

Comment:
could you check how the naming styling compares to what's in this doc:
https://protobuf.dev/programming-guides/style/

Reply:
Looks correct to me (pascalcase)